hvpy/io.py

```diff
@@ -1,11 +1,12 @@
+import os
 from enum import Enum, auto
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 from pydantic import BaseModel
 
-__all__ = ["HvpyParameters", "OutputType"]
+__all__ = ["HvpyParameters", "OutputType", "set_api"]
 
-BASE_URL = "https://api.helioviewer.org/v2/"
+base_url = "https://api.helioviewer.org/v2/"
```

Comment:
```suggestion
if "URL" in os.environ:
    base_url = os.environ.get("URL")
else:
    base_url = "https://api.helioviewer.org/v2/"
```

So will it be like this?

Reply:
We can set the second argument to `get` to be the main URL and we won't need the if statement.

It should also be renamed to something other than `URL`, its too vague.